pkg/registry/manifest/manifest.go

```diff
@@ -53,7 +53,7 @@ func extractImageAndTag(imageName string) (string, string) {
 		parts := strings.Split(imageName, ":")
 		if len(parts) > 2 {
 			img = fmt.Sprintf("%s%s", parts[0], parts[1])
```

Comment:
```suggestion
			img = fmt.Sprintf("%s:%s", parts[0], parts[1])
```

Reply:
Unfortunately, this won't resolve the issue in #730 either, as it will disregard the hash pinning.